engine/src/main/resources/settings.ini

```diff
@@ -2,8 +2,8 @@ x=1366
 y=768
 fullscreen=false
 controlType=1
-sfxVol=1.0
-musicVol=0.25
+sfxVol=0.0
```

Comment:
Please don't commit your settings :)

Reply:
Definitely afree with Malanius - please revert this file.

Reply:
Sorry, that was an accident!